File: app/src/main/java/com/tonitealive/app/ui/views/signup/SignUpActivity.java
package com.tonitealive.app.ui.views.signup;

import android.support.v4.app.Fragment;

import com.tonitealive.app.ui.views.SingleFragmentActivity;


public final class SignUpActivity extends SingleFragmentActivity {

    @Override
    protected Fragment createFragment() {
        return SignUpFragment.newInstance();
    }

}

File: app/src/main/kotlin/com/tonitealive/app/ui/views/signup/SignUpActivity.kt
package com.tonitealive.app.ui.views.signup

import android.support.v4.app.Fragment
import com.tonitealive.app.ui.views.SingleFragmentActivity


class SignUpActivity : SingleFragmentActivity() {

    override fun createFragment(): Fragment {
        return SignUpFragment.newInstance()
    }
}